src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -5,7 +5,7 @@ import { List } from 'react-virtualized';
 import Measure from 'react-measure';
 import classNames from 'classnames';
 
-import Thumbnail from 'components/Thumbnail';
+import Thumbnail, { THUMBNAIL_SIZE } from 'components/Thumbnail';
```

Comment:
`THUMBNAIL_SIZE ` is null and `getNumberOfColumns` breaks because it comparing with undefined values.  I think in `components/Thumbnail/index.js`, you need to change it to 
```
export { default } from './Thumbnail';
export { THUMBNAIL_SIZE } from './Thumbnail';
```